odl/tomo/geometry/geometry.py

```diff
@@ -164,26 +164,6 @@ def det_refpoint(self, mpar):
             The reference point, an `ndim`-dimensional vector
         """
 
-    @abstractmethod
```

Comment:
Not quite sure i agree on removing it, by the same argument we could remove the `motion_params` as well, no?